pkg/netid/netid.go

```diff
@@ -0,0 +1,61 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Current assigned VNID for the namespace
+	VNIDAnnotation string = "pod.network.openshift.io/multitenant.vnid"
+	// Desired VNID for the namespace
+	WantsVNIDAnnotation string = "pod.network.openshift.io/multitenant.wants-vnid"
```

Comment:
My intent was to convey this annotation is for pod network in openshift and it holds VNID when multitenant network plugin is used. 
'cluster' doesn't sound right...
@smarterclayton do you have any suggestions?